### ISSUE TITLE
5200: sd5200.set

### ISSUE THREAD

**Issue by tjanas** [frederic-mahe/Hardware-Target-Game-Database#583]
I believe this is a character set file for the UI of the Atari 5200 Ultimate SD Cartridge flash cart, and should probably be removed from the [database](https://github.com/frederic-mahe/Hardware-Target-Game-Database/blob/master/EverDrive%20Pack%20SMDBs/Atari%205200%20SMDB.txt).
`4917edd5d10f87540a58d2fe93bfc792e4751f17b24119c23f44720f43ab29a4	Atari 5200/sd5200.set`

**Comment by Cavia** [comment]
Yes, and I would also suggest to remove the following `Atari 5200\4 Hacks & Homebrew\Pac-Man (throttle hack).atr` as I'm not even sure is is a valid Atari 5200 game format.
